Sources/UINotificationView.swift

```diff
@@ -71,6 +85,13 @@ open class UINotificationView: UIView {
         return imageView
     }()
     
+    var button: UIButton? {
+        let button = self.notification.button
```

Comment:
We could also decide to work with a `UINotificationAction` again and set that as a property on `notification`. This way we don't have to pass a `UIButton` instance from the notification, which is kind of strange to do if you ask me.

Also, we keep the API in sync with the code for adding an action to the tap of a notification.

After that, we can make this button private, having a callback in this view and allow customization through subclassing the `UINotificationView` just like we do for other logic.

Let me know your thoughts!

Reply:
I just mimicked Apple's implementation of `UINavigationItem` with the `rightBarButtonItem` [property](https://developer.apple.com/documentation/uikit/uinavigationitem/1624957-rightbarbuttonitem?language=objc).
My rational was that most iOS developers are familiar with this pattern and it will therefore make it easier for them to work with our API.

Let me know if you feel strongly about it and I'd be happy to change the implementation to what you describe.

Reply:
Ah, that makes sense!

The thing I don't like is that we're passing a `UIButton` with a view model kind of `Notification`. As if we're setting up a `rightBarButtonItem` through a `UIButton` that comes from a view model.

I guess it's a bit hard to compare. Let's keep it like it is, I don't feel too strongly about it.

I would like to see documentation for this button though! And can we keep it private?